src/lazy/expanded/macro_evaluator.rs

```diff
@@ -196,12 +186,12 @@ impl<'top, D: Decoder> MacroExpr<'top, D> {
         MacroExprArgsIterator { source: args_kind }
     }
 
-    pub(crate) fn invoked_macro(&self) -> MacroRef<'top> {
+    pub(crate) fn invoked_macro(&self) -> &'top Macro {
```

Comment:
🗺️ Previously, the system required that all `MacroExpr`s could tell you their address in the macro table. However, it is now possible for a macro's body to invoke a macro that is not addressable via the active encoding context. This required changing several field types from `MacroRef<'top>` to `&'top Macro`. (`&'top Macro` is only 8 bytes compared to `MacroRef`'s 16 bytes though, so it's actually kind of nice.)